### ISSUE TITLE
Content is unable to drag and drop by using keyboard.

### ISSUE THREAD

**Issue by msftedad** [tiberiuzuld/angular-gridster2#876]
**Description:** Keyboard users will be affected if they are not able to drag and drop the items using keyboard.

**Test environment:**
OS: Windows_11 
Version: 22H2
OS Build: 22621.1265
Browser: edge 110.0.1587.63 (Official build) (64-bit)
URL: https://tiberiuzuld.github.io/angular-gridster2/

**Repro Steps:**
1) Open URL: https://tiberiuzuld.github.io/angular-gridster2/
2) Make sure 'Home' tab is selected.
3) Navigate using tab key and press enter key to activate 'Arrange tiles' where popup opens.
4) Try to drag and drop by using keyboard.
5) Observe an issue that content is unable to drag and drop by using keyboard.

**Actual Result:** 
Content is not draggable by using keyboard.

**Expected Result:**
Control(s) must be accessible to keyboards and other assistive technology. Common causes of this problem include a) the element does not have a proper role assigned, b) the element needs tab-index="0" attribute to be focusable, or c) the component is not registering keypresses.

https://user-images.githubusercontent.com/93735775/225546330-be54292d-0e85-427b-b360-ac0675f48632.mp4

**Comment by msftedad** [comment]
Hi Team , any update on this?

**Comment by msftedad** [comment]
Hi Team , any update on this bug?

**Comment by msftedad** [comment]
Hi Team,

Any update on this issue?

**Comment by tiberiuzuld** [comment]
Hello,

I accept PRs
If you want this feature make a PR with it.

I don't have the time to implement other people's wishes. This goes for all open issues here.

Have a nice day.
